dist_ffi/dist_ffi.go

```diff
@@ -149,8 +149,8 @@ func listenOnSocket(l net.Listener, c chan MsgAndSender) {
 	for {
 		conn, err := l.Accept()
 		if err != nil {
-			// Assume() no error on Listen
-			panic(err)
+			// Some client connected but that did not work out... they better retry.
+			continue
```

Comment:
I'm not about the exact situations that cause an error to be sent by `Accept`; I've not observed any situations where it happens, and I'd be more worried about something getting messed up with the listening socket, in which case retrying `Accept` will just be spinning. I think I'd rather keep this panic here until we run into a situation where we don't want it to panic.

Reply:
Fair, I took back this change.